### PR TITLE
Use the ImagePlaceholder component on OnDemand radio

### LINKED ISSUE #7097: Use image-with-placeholder for hero image on OD radio
**Describe the bug**

The hero is missing on OD Radio AMP pages when JavaScript is disabled.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/indonesia/bbc_indonesian_radio/programmes/w13xtt0s.amp
2. Disable JavaScript
3. Note the missing hero image

**Expected behaviour**
The hero should show as it does on canonical pages

**Screenshots**

image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves #7097

**Overall change:**
- Replacing the current Image component in OnDemand radio with the `ImagePlaceholder` component
- We were previously passing the `srcset` down to both canonical & amp images on OD radio, the `ImagePlaceholder` component wasn't set up to pass `srcset` to `<amp-img>` (which seems like a totally valid thing to do - https://amp.dev/documentation/components/amp-img/), so I'm now passing that prop down. 

**Code changes:**

- Use `ImagePlaceholder` on OnDemand radio
- Pass `secset` prop to AMP img in `ImagePlaceholder`
- Update snaps

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)

For testers, this is fixing a niche bug where you have JS disabled on an AMP page, and the image won't render (as amp-img relies on js). Instead, we are now rendering this placeholder.

The PR has the added benefit of adding the placeholder on canonical, too - improving perceived loading performence:

Screenshot 2020-08-28 at 15 45 47

### CODE REVIEW

File: src/app/containers/OnDemandImage/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
 
 exports[`AudioPlayer blocks OnDemandHeading should render correctly 1`] = `
-.c1 {
+.c2 {
   display: block;
   width: 100%;
   visibility: visible;
 }
 
+.c1 {
```
Comment (RichardPK) [inline src/app/containers/OnDemandImage/__snapshots__/index.test.jsx.snap:10]: This is the new placeholder image

File: src/integration/pages/mediaAssetPage/pidgin/__snapshots__/canonical.test.js.snap [hunk 2]
```diff
@@ -133,24 +133,14 @@ exports[`Canonical Media Asset Page I can see a bulleted list item 1`] = `"<Medi
 
 exports[`Canonical Media Asset Page I can see a bulleted list item with link: <Media Asset Page> 1`] = `"http://www.bbc.com/afrique/23248423"`;
 
-exports[`Canonical Media Asset Page I can see related content: 'I no know say I different for society until pipo begin look me one kain' 1`] = `"/pidgin/media-53580248"`;
```
Comment (RichardPK) [inline src/integration/pages/mediaAssetPage/pidgin/__snapshots__/canonical.test.js.snap:136]: Ffs I think this is coming from updating integration snapshots that are failing on latest. When this is fixed on latest, it'll disappear here.

File: src/app/pages/FrontPage/__snapshots__/index.test.jsx.snap [hunk 3]
```diff
@@ -937,6 +937,7 @@ exports[`Front Page snapshots should render a pidgin amp frontpage 1`] = `
                                 height="549"
                                 layout="responsive"
                                 src="https://ichef.bbci.co.uk/news/660/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg"
+                                srcset="https://ichef.bbci.co.uk/news/70/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 320w, https://ichef.bbci.co.uk/news/660/cpsprodpb/16D8E/production/_107328539_ninthnationalassemblypidgin_senate_reps_1080.jpg 660w"
```
Comment (RichardPK) [inline src/app/pages/FrontPage/__snapshots__/index.test.jsx.snap:940]: These are being updated because we're now passing `srcset` to the AMP image

**Review by andrewscfc — APPROVED** [review]
I see no issues with the snapshots you've updated for Stories and Frontpages. PR looks good 👍

**Review by karinathomasbbc — APPROVED** [review]

**Pull request merged**